examples/step-3/step-3.cc

```diff
@@ -167,7 +167,8 @@ void Step3::make_grid()
   GridGenerator::hyper_cube(triangulation, -1, 1);
   triangulation.refine_global(5);
 
-  std::cout << "Number of active cells: " << triangulation.n_active_cells()
+  std::cout << "Number of active cells: "     //
+            << triangulation.n_active_cells() //
```

Comment:
I find these bare `//` comments pretty confusing. What's a user supposed to make of them?

Reply:
i don't think they are necessary here.